CYCLONEDDS_QUALITY_DECLARATION.md

```diff
@@ -14,65 +14,90 @@ Below are the rationales, notes, and caveats for this claim, organized by each r
 
 ### Version Stability [1.ii]
 
-The latest release of `CycloneDDS` is considered stable, but there is no documentation about whether all future releases should be considered stable.
+`CycloneDDS` is at a stable version. The current version can be found in its [package.xml](https://github.com/eclipse-cyclonedds/cyclonedds/blob/master/package.xml), and its change history can be found in its [CHANGELOG](https://github.com/eclipse-cyclonedds/cyclonedds/blob/master/CHANGELOG.rst).
 
 ### Public API Declaration [1.iii]
 
-The public API for `CycloneDDS` is made up of symbols defined in the header files it installs.
-In the source repository, these header files reside with each of the modules that make up `CycloneDDS`.
+All symbols in the installed headers are considered part of the public API. In the source repository, these header files reside with each of the modules that make up CycloneDDS.
 
 ### API Stability Policy [1.iv]
 
-There is no published policy for API stability in `CycloneDDS`, however there is some information in the [Releases section](https://www.eclipse.org/projects/handbook/#release) of the Eclipse project handbook which states that releases which include breaking API changes are considered "Major".
+`CycloneDDS` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released. It conforms to the [Releases section](https://www.eclipse.org/projects/handbook/#release) of the Eclipse project handbook which states that releases which include breaking API changes are considered "Major".
 
 ### ABI Stability Policy [1.v]
 
-There is no published policy for ABI stability in `CycloneDDS`.
+`CycloneDDS` provides ABI stability for PATCH release. `CycloneDDS` strives to provide ABI stability for MINOR release. `CycloneDDS` not guarantee ABI stability for MAJOR release.
 
 ### ABI and ABI Stability Within a Released ROS Distribution [1.vi]
 
+`CycloneDDS` will not break API nor ABI within a released ROS distribution, i.e. no major releases once the ROS distribution is released. 
 As an external package, `CycloneDDS` is not released on the same cadence as ROS.
 Changes to the branch of the `CycloneDDS` repository that is targeted by a given ROS release would be picked up by development builds of that ROS release.
-There is no published policy for what criteria are required for changes that are accepted in those branches.
 
-## Change Control Process [2]
+## Change Control [2]
+
+`CycloneDDS` follows the recommended guidelines of the [Eclipse Development Process](https://www.eclipse.org/projects/dev_process/).
+
+For the RMW layer, we follow the ROS core packages process. For Eclipse Cyclone DDS, we ensure the stability within the project through review, CI and tests and additionally run ROS CI for changes that are likely to affect ROS.
```

Comment:
GIven what is stated in line 1 of the document, the first sentence of this paragraph is superfluous.